mobile_verifier/src/rewarder.rs

```diff
@@ -301,12 +305,21 @@ where
         transaction.commit().await?;
 
         // now that the db has been purged, safe to write out the manifest
+        let reward_data = ManifestMobileRewardData {
+            poc_bones_per_coverage_point: Some(helium_proto::Decimal {
+                value: poc_dc_shares.normal.to_string(),
```

Comment:
Do we want to support a maximum amount precision?

The [rust_decimal docs](https://docs.rs/rust_decimal/latest/rust_decimal/index.html) says trailing zeros may be exposed when in string form, do we want to remove those?

Reply:
I don't think that's an issue personally

Reply:
To more precisely address this, here is what the google decimal specification says (which we are adhering to):
```
 Additionally, services **may** preserve trailing zeroes in the fraction to indicate increased precision, but are not required to do so.
 ```
so either option is fine